src/utils/initialData.ts

```diff
@@ -0,0 +1,25 @@
+const initialPokemonCardData = {
```

Comment:
I wouldn't pre-populate this with non-api data, just because the url or something could be out-of-date. If you want initial data, I'd make an initial API call or wait for your existing calls to resolve.